### PR TITLE
Fix UCSC hg38 link

### LINKED ISSUE #5118: UCSC link from RNAfusion (hg38) pages
**Describe the bug**
UCSC links from RNAfusion pages don't work. From a Quick Look, it kind of seems this will be global for hg38 UCSC links. Might be us not testing, might be them updating to a more sensible versioning. 

**To Reproduce**
1. Go to an RNAfusion case
2. Click on UCSC
3. See error

**Screenshots**
Screenshot 2024-12-12 at 09 52 13

Changing "hg20" for "hg38":
Screenshot 2024-12-12 at 09 53 18

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

- Fix #5118 

Screenshot 2024-12-12 at 10 02 58

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by DN

### CODE REVIEW

File: scout/server/links.py [hunk 1]
```diff
@@ -714,7 +714,7 @@ def ucsc_link(variant_obj, build=None):
     )
     if build == 38:
         url_template = (
-            "http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg20&"
+            "http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&"
```
Comment (northwestwitch) [inline scout/server/links.py:717]: 20? weird!

**Review by northwestwitch — COMMENTED** [review]
👍🏻

File: scout/server/links.py (hunk 1 above)
Comment (dnil) [reply scout/server/links.py:717]: I know! Very much wonder if it was ever like that on their side or if it is our typo…

**Comment by dnil** [comment]
I am pretty sure it was them: influential parts at ucsc really wanted to stick to their old version series instead of switching to sync up with GRC. Probably both worked for a long time.

IMG_9084

**Review by northwestwitch — APPROVED** [review]
Ah sorry I forgot to choose approve apparently!

**Pull request merged**